sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -36,3 +36,5 @@ select date '2001-10-01' - 7;
 select date '2001-10-01' - date '2001-09-28';
 select date'2020-01-01' - timestamp'2019-10-06 10:11:12.345678';
 select timestamp'2019-10-06 10:11:12.345678' - date'2020-01-01';
+select date '2001-09-28' + integer '7';
```

Comment:
I don't think this is necessary test. `datetime.sql` aims to test date-time related functionality. And `date +/- integer` has been already covered by https://github.com/apache/spark/pull/26291/files#diff-e7a79cbcd5411e8891cee65c286c2177L33

Reply:
thanks for your review. updated